src/test/fixtures/profiles/processed-profile.js

```diff
@@ -657,3 +659,31 @@ export function getProfileWithJsTracerEvents(
   );
   return profile;
 }
+
+export function getCounter(
+  thread: Thread,
+  mainThreadIndex: ThreadIndex,
+  countRange: number
+): Counter {
+  const counter: Counter = {
+    name: 'My Counter',
+    category: 'My Category',
+    description: 'My Description',
+    pid: thread.pid,
+    mainThreadIndex,
+    sampleGroups: {
+      id: 0,
+      samples: {
+        time: thread.samples.time.slice(),
+        // Create some arbitrary values for the number.
+        number: thread.samples.time.map((_, i) =>
+          Math.floor(50 * Math.sin(i) + 50)
+        ),
+        // Create some arbitrary (positive integer) values for the count.
```

Comment:
I believe the comment isn't right, as this isn't an integer, and this can be negative.
Maybe you inverted the 2 comments for `count` and `number`, as this comment seems right for `number`, and the other comment seems right for `count` :)